recipes/kaldi/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: kaldi
+  version: r7271.1a4dbf6
+
+source:
+  git_url: https://github.com/kaldi-asr/kaldi.git
```

Comment:
Please use the tarball format. See https://github.com/conda-forge/nco-feedstock/blob/master/recipe/meta.yaml#L8-L10 for an example.